<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevAdd</class>
 <widget class="QDialog" name="DevAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string/>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>287</width>
       <height>246</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,1,1,1,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_52">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>端口号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Source Han Sans CN</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
    border: 1px solid #4C84C3;
	width: 181px;
	height: 40px;
	font-size: 15px;
	font-family: Source Han Sans CN;
	background-color: rgb(211, 215, 207);
	font-weight: 400;
	opacity: 1;
	border-radius: 0px;
}
 
QComboBox QAbstractItemView {
    border-radius:  0px;  
	background-color: rgb(233, 185, 110);
	border:1px solid #4C84c3;
 	selection-background-color: #2019DC;
}
 
QComboBox QAbstractItemView::item {
    height: 40px;   
	border: none;
}
 
</string>
          </property>
          <item>
           <property name="text">
            <string>COM1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>USB-COM1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>USB-COM2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>USB-COM3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>USB-COM4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>XR-COM1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>XR-COM2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>XR-COM3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>XR-COM4</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_55">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>波特率：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_baud">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Source Han Sans CN</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
    border: 1px solid #4C84C3;
	width: 181px;
	height: 40px;
	font-size: 15px;
	font-family: Source Han Sans CN;
	background-color: rgb(211, 215, 207);
	font-weight: 400;
	opacity: 1;
	border-radius: 0px;
}
 
QComboBox QAbstractItemView {
    border-radius:  0px;  
	background-color: rgb(233, 185, 110);
	border:1px solid #4C84c3;
 	selection-background-color: #2019DC;
}
 
QComboBox QAbstractItemView::item {
    height: 40px;   
	border: none;
}
 
</string>
          </property>
          <item>
           <property name="text">
            <string>1200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_56">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>停止位：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_stopbit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Source Han Sans CN</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
    border: 1px solid #4C84C3;
	width: 181px;
	height: 40px;
	font-size: 15px;
	font-family: Source Han Sans CN;
	background-color: rgb(211, 215, 207);
	font-weight: 400;
	opacity: 1;
	border-radius: 0px;
}
 
QComboBox QAbstractItemView {
    border-radius:  0px;  
	background-color: rgb(233, 185, 110);
	border:1px solid #4C84c3;
 	selection-background-color: #2019DC;
}
 
QComboBox QAbstractItemView::item {
    height: 40px;   
	border: none;
}
 
</string>
          </property>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_35" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_60">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>校验位：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_parity">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Source Han Sans CN</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
    border: 1px solid #4C84C3;
	width: 181px;
	height: 40px;
	font-size: 15px;
	font-family: Source Han Sans CN;
	background-color: rgb(211, 215, 207);
	font-weight: 400;
	opacity: 1;
	border-radius: 0px;
}
 
QComboBox QAbstractItemView {
    border-radius:  0px;  
	background-color: rgb(233, 185, 110);
	border:1px solid #4C84c3;
 	selection-background-color: #2019DC;
}
 
QComboBox QAbstractItemView::item {
    height: 40px;   
	border: none;
}
 
</string>
          </property>
          <item>
           <property name="text">
            <string>N</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>E</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>O</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_57">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>数据位：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_databit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Source Han Sans CN</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
    border: 1px solid #4C84C3;
	width: 181px;
	height: 40px;
	font-size: 15px;
	font-family: Source Han Sans CN;
	background-color: rgb(211, 215, 207);
	font-weight: 400;
	opacity: 1;
	border-radius: 0px;
}
 
QComboBox QAbstractItemView {
    border-radius:  0px;  
	background-color: rgb(233, 185, 110);
	border:1px solid #4C84c3;
 	selection-background-color: #2019DC;
}
 
QComboBox QAbstractItemView::item {
    height: 40px;   
	border: none;
}
 
</string>
          </property>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string/>
    </attribute>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>331</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_24">
      <item>
       <widget class="QLabel" name="label_58">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>服务地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ipAddr">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(211, 215, 207);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_59">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>371</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>注：服务地址为空，则为串口设备，否则为网口设备。</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>811</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="title">
    <string>设备参数</string>
   </property>
   <widget class="QTextEdit" name="textEdit_devParams">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>771</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>381</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="title">
    <string>设备基本配置</string>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>319</width>
      <height>167</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,1,1,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_33" stretch="2,5">
       <item>
        <widget class="QLabel" name="label_49">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>接口类型：</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QRadioButton" name="radioButton_Com">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>串口设备</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_Net">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>网口设备</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_32" stretch="2,5">
       <item>
        <widget class="QLabel" name="label_54">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>设备地址：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_devAddr">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(211, 215, 207);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_31" stretch="2,5">
       <item>
        <widget class="QLabel" name="label_53">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>设备类型：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Source Han Sans CN</family>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox {
    border: 1px solid #4C84C3;
	width: 181px;
	height: 40px;
	font-size: 15px;
	font-family: Source Han Sans CN;
	background-color: rgb(211, 215, 207);
	font-weight: 400;
	opacity: 1;
	border-radius: 0px;
}
 
QComboBox QAbstractItemView {
    border-radius:  0px;  
	background-color: rgb(233, 185, 110);
	border:1px solid #4C84c3;
 	selection-background-color: #2019DC;
}
 
QComboBox QAbstractItemView::item {
    height: 40px;   
	border: none;
}
 
</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_36" stretch="2,5">
       <item>
        <widget class="QLabel" name="label_devName">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>设备名称：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(211, 215, 207);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>550</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>550</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
